tool/admin/release/index.html

```diff
@@ -209,42 +209,42 @@ <h2>Website Tests</h2>
           <h2>Website Lib</h2>
```

Comment:
I'd vote for dropping the admin stuff and most parts of the tech documentation before 6.0. It does not get deployed and the technical information in there is outdated anyways.

Reply:
:+1

Reply:
+1 but we should have some technical information somewhere. It was painfull to figure this out. And it‘s not very obvious how you can make a lokal build to test the changes in the build toolchain

Reply:
But this is another sory.